### ISSUE TITLE
some useragents are permanently banned from cloudflare

### ISSUE THREAD

**Issue by Revilum** [gigobyte/HLTV#450]
I had some issues where I would get banned by cloudflare way too often but i couldn't reproduce the issue. I heavily reduced the amount of requests but the issue was still occurring and the amount of requests being banned was the same. Even after some stress tests i found out that spamming one page still took a long time until I was banned so I had one final idea: Since the release of v3, random useragents are used getting data from hltv so this might be the cause of the issue as this is the only other random thing that could influence the random bans from cloudflare. After installing [this](https://addons.mozilla.org/en-US/firefox/addon/user-agent-string-switcher/) addon for firefox and selecting 'Internet Explorer' 6.0 as my useragent. When visiting [hltv](https://hltv.org) I'll get the following error message: https://i.imgur.com/oFuw1nH.png
After some research I found [this](https://support.cloudflare.com/hc/en-us/articles/115001856951-Understanding-Cloudflare-User-Agent-Blocking) article.

**Comment by gigobyte** [comment]
Wow, thanks for the info! That's really useful. I'll try to configure the random user agent library so that it doesn't return IE.

**Comment by Revilum** [comment]
> Wow, thanks for the info! That's really useful. I'll try to configure the random user agent library so that it doesn't return IE.

it's not just IE. other versions work too so my guess is that there is a pattern on what is banned or it was randomly chosen by HLTV.

**Comment by Revilum** [comment]
After some testing I made [this](https://github.com/gigobyte/HLTV/files/6272684/not-working.txt) list of useragents not working. The only thing I could find out was the all IE v6 are banned so `MSIE 6.0` in any way causes the issue. There may be some more troublesome agents because I managed to find 4 agents in 100 agents that were banned which way less than the api has this problem. In v2 the problem was occurring or at least not as bad so that might be a place to start.
